core/email.py

```diff
@@ -239,11 +239,9 @@ def email_to_admin(
             user_email = "%s@%s" % (username, settings.DEFAULT_EMAIL_DOMAIN)
     elif not username:  # user_email provided
         username = 'Unknown'
-    if request_tracker or not cc_user:
-        # Send w/o the CC
-        cc = []
-    else:
-        cc = [email_address_str(username, user_email)]
+    cc = None
+    if not request_tracker and cc_user:
+        cc = email_address_str(username, user_email)
```

Comment:
I'm not sure what the problem in ATMO-2169 is then since it is already CC'ing a list of one email and the ADMINS_EMAIL_TUPLE variable is not used in Atmosphere.

Reply:
ADMINS_EMAIL_TUPLE is used in Atmosphere. The ticket's purpose (unclear) was to make things more reasonable. Currently on j7m prod we configure that variable with a list of admins emails, but only the first is used. We don't strictly need the multiple admins there, so the ticket was open to the option of either supporting a list of admins or a single admin, but that the variables needed to reflect the actual use in the code.

Reply:
Oh I see, ADMINS_EMAIL_TUPLE is used to define the ADMINS variable.